detox/src/devices/drivers/android/genycloud/exec/GenyCloudExec.js

```diff
@@ -0,0 +1,45 @@
+const _ = require('lodash');
+const exec = require('../../../../../utils/exec').execWithRetriesAndLogs;
+
+class GenyCloudExec {
+  getRecipe(name) {
```

Comment:
I would suggest adding a check:

```
➜  test git:(genymotion-cloud) ✗ gmsaas auth whoami
No credentials found. To set them up, use 'gmsaas auth login' command.
```

If we see something different from a login or email address, this is the right time to fail early with an error.

Reply:
Ah, I see. It fails with this error anyway. No worries then. 👍